resources/translations/pl.po

```diff
@@ -2691,16 +2691,6 @@ msgstr ""
 "Uruchom na pełnym ekranie (domyślnie wyłączone).\n"
 "Skróty klawiszowe do zarządzania oknem można wyświetlić poleceniem 'intro'."
 
-#: src/config/setup.cpp:274
-#, fuzzy, c-format, no-wrap
-msgctxt "CONFIG_FULLRESOLUTION"
-msgid ""
-"The [color=light-green]'fullresolution'[reset] setting is deprecated but still accepted;\n"
-"please use [color=light-green]'fullscreen_mode'[reset] instead."
-msgstr ""
-"Rozdzielczość pełnego ekranu: 'original', 'desktop' (domyślne) lub konkretna\n"
-"rozdzielczość, np. 1024x768."
-
```

Comment:
Not strictly necessary to remove this from the `.po` files as those will need to be regenerated anyway to sync them up with the code, but I guess it doesn't hurt in this case.

Reply:
This might answer your questions:
https://github.com/dosbox-staging/dosbox-staging/blob/main/docs/TRANSLATING.md

If not, ask @FeralChild64 😄 

I never paid much attention to translations. Previously, we always left them in a broken or semi-broken state until the guy maintaining the translations for a particular language did the syncing/updating (usually a few weeks or a month before the release, so yeah, we should give them a heads-up before cutting the RC). From a high-level perspective, that's not gonna change, otherwise we'd be constantly bottlenecked by translator availability whenever we change any message. That's simply not practical.

Reply:
I left the translation files as-is in my latest push.

Reply:
Thanks @FeralChild64 , that's useful info to keep in mind.